### PR TITLE
add detached mode to migration controller

### PR DESCRIPTION
**Pull request by xuezhaojun**
Signed-off-by: xuezhaojun <zxue@redhat.com>

https://github.com/open-cluster-management-io/registration-operator/issues/157

### CODE REVIEW

**Comment by xuezhaojun** [comment]
/assign @zhujian7 
/assign @zhiweiyin318 
PTAL

File: pkg/helpers/helpers.go [hunk 1]
```diff
@@ -663,3 +664,34 @@ func SyncSecret(client, targetClient coreclientv1.SecretsGetter, recorder events
 		return resourceapply.ApplySecret(targetClient, recorder, source)
 	}
 }
+
+func GetHubKubeconfig(ctx context.Context,
```
Comment (qiujian16) [inline pkg/helpers/helpers.go:671]: this name is confusing, and please add more comment on this

**Review by qiujian16 — COMMENTED** [review]
/approve

please rename the func

File: pkg/helpers/helpers.go (hunk 1 above)
Comment (xuezhaojun) [reply pkg/helpers/helpers.go:671]: Yes, comment added and also change the argument from `kubeconfig` to `managementKubeconfig`.

**Pull request merged**